vcstool/commands/import_.py

```diff
@@ -36,12 +41,28 @@ def __init__(
         self.shallow = shallow
 
 
+def file_or_url_type(x):
+    if x.startswith(('http://', 'https://')):
```

Comment:
Instead of explicitly whitelisting schemes how about inverting the logic and check if the passed value is a valid file path?

Reply:
by "valid" do you mean "existing"?

If so, then if the file does not exist, it will try using the path as a URL, and it will print an HTTP/URL-related error, i.e. it will never say that the file just doesn't exist.

Reply:
> by "valid" do you mean "exists"?

Yes, that was my idea.

> If so, then if the file does not exist, it will try using the path as a URL, and it will print an HTTP/URL-related error because it's not really a URL, i.e. it will never say that the file just doesn't exist.

Maybe before trying to load the request it can try to parse the value first and check that it has a scheme part?

Reply:
see fef2e82d3f3353c2c7fccd04ef4bdfdf5a4ca771, I think it's a good middle ground